components/avatar.css

```diff
@@ -0,0 +1,47 @@
+.avatar-round-sm,
+.avatar-sq-sm {
+    width: 3.12rem;
+    height: 3.12rem;
+    font-size: 1.2rem;
+}
```

Comment:
This can be refactored into 

```
.avatar-sm { 
    width: 3.12rem;
    height: 3.12rem;
    font-size: 1.2rem;
}

.avatar-sq {
    border-radius: 20%
}

```

Doing this way you are writing less code than your current approach and it is more objective in nature.

Reply:
Thank you for suggesting this, actually by my own personal preference instead of using .avatar-sm , .avatar-sq ,  I prefer directly using avatar-sq-sum class as I  have used it in other component also which I  have already merged

Reply:
Cool